apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -5,550 +5,656 @@ const { createTestContact } = require('@condo/domains/contact/utils/testSchema')
-        const admin = await makeLoggedInAdminClient()
-        const { employeeFrom, clientFrom, organizationFrom, propertyFrom } = await createTestOrganizationWithAccessToAnotherOrganization()
-        await createTestTicket(admin, organizationFrom, propertyFrom)
-        await updateTestOrganizationEmployee(admin, employeeFrom.id, {
-            deletedAt: 'true',
-        })
+        test('deleted user: cannot read "to" tickets', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const {
+                employeeFrom,
+                clientFrom,
+                organizationFrom,
+                propertyFrom,
+            } = await createTestOrganizationWithAccessToAnotherOrganization()
+            await createTestTicket(admin, organizationFrom, propertyFrom)
+            await updateTestOrganizationEmployee(admin, employeeFrom.id, {
+                deletedAt: 'true',
+            })
 
-        const tickets = await Ticket.getAll(clientFrom)
-        expect(tickets).toHaveLength(0)
-    })
+            const tickets = await Ticket.getAll(clientFrom)
+            expect(tickets).toHaveLength(0)
+        })
+
+        test('blocked user: cannot create "to" tickets', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const {
+                employeeFrom,
+                clientFrom,
+                organizationTo,
+                propertyTo,
+                organizationFrom,
+            } = await createTestOrganizationWithAccessToAnotherOrganization()
+            const [role] = await createTestOrganizationEmployeeRole(admin, organizationFrom, {
+                canManageTickets: true,
+            })
+            await updateTestOrganizationEmployee(admin, employeeFrom.id, {
+                role: { connect: { id: role.id } },
+            })
+            await createTestTicket(clientFrom, organizationTo, propertyTo)
+            await updateTestOrganizationEmployee(admin, employeeFrom.id, {
+                isBlocked: true,
+            })
 
-    test('blocked user: cannot create "to" tickets', async () => {
-        const admin = await makeLoggedInAdminClient()
-        const { employeeFrom, clientFrom, organizationTo, propertyTo, organizationFrom } = await createTestOrganizationWithAccessToAnotherOrganization()
-        const [role] = await createTestOrganizationEmployeeRole(admin, organizationFrom, {
-            canManageTickets: true,
-        })
-        await updateTestOrganizationEmployee(admin, employeeFrom.id, {
-            role: { connect: { id: role.id } },
-        })
-        await createTestTicket(clientFrom, organizationTo, propertyTo)
-        await updateTestOrganizationEmployee(admin, employeeFrom.id, {
-            isBlocked: true,
+            await expectToThrowAccessDeniedErrorToObj(async () => {
+                await createTestTicket(clientFrom, organizationTo, propertyTo)
+            })
         })
+    })
```

Comment:
Do you have a test for "property soft delete case"?

Reply:
Now I do 😃